webui/react/src/components/kit/LineChart.tsx

```diff
@@ -58,6 +61,8 @@ interface Props {
   yLabel?: string;
 }
 
+export type ChartProps = Props[];
```

Comment:
I'm open to this in this file if:
- it's used in `GroupProps` (and related jsdoc) later in this file where it is currently `chartsProps: Props[]`
- the name conveys that it's being used for multiple charts (ChartsProps, ChartGridProps, or MultiChartProps)